force-app/main/default/classes/NoteController.cls

```diff
@@ -1,39 +1,82 @@
 public with sharing class NoteController {
     @AuraEnabled
-    public static Id saveData(String label, String description ) {
+    public static Note__c saveData(String label, String description ) {
         try {
             
             Note__c newNote = new Note__c();
             newNote.Label__c = label;
             newNote.Description__c = description;
-            newNote.Created_Date__c = Date.today();
+            newNote.Created_Date__c = Datetime.now();
     
-            insert newNote;             
-            return newNote.Id;
+            insert newNote;
+
+            return newNote;
         } catch (Exception e) {
             System.debug('Error saving data: ' + e.getMessage());
             return null;
```

Comment:
I recommend to throw AuraHandledException here and you can catch it in front in your component:
```
callToApex({paramName: someValue})
            .then((result) => {})
            .catch(error) {
                //handle your AuraHandledException here;
            }
        ```